UI.tsv

```diff
@@ -614,7 +614,7 @@ UI_20150317_000613	Rest Quickslot
 UI_20150317_000614	{@st43}Dialog box{/}
 UI_20150317_000615	Temporary Name
 UI_20150317_000616	{@st41b} {s18} lack of inventory space
-UI_20150317_000617	Inventory is full, you can not obtain the item, please arrange the {nl} inventory.
+UI_20150317_000617	Inventory is full, you can not carry the item {nl}please re-arrange your inventory.
```

Comment:
Try to leave the {nl} tags in the original place. Most of the time they are put in a certain place in the sentence because of the in-game ui.